semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/SymbolOps.scala

```diff
@@ -106,4 +96,18 @@ trait SymbolOps { self: DatabaseOps =>
       }
     }
   }
+
+  implicit class XtensionGSymbolMCategories(sym: g.Symbol) {
+    def isSemanticdbGlobal: Boolean = !isSemanticdbLocal
+    def isSemanticdbLocal: Boolean = {
+      def definitelyGlobal = sym.hasPackageFlag
+      def definitelyLocal =
+        sym == g.NoSymbol ||
+          sym.name.decoded.startsWith(g.nme.LOCALDUMMY_PREFIX) ||
+          (sym.owner.isMethod && !sym.isParameter) ||
+          ((sym.owner.isAliasType || sym.owner.isAbstractType) && !sym.isParameter)
```

Comment:
@olafurpg For a short period of time, parameters and type parameters of global methods and types were considered local. In this pull request, I had to revert that design decision and make these symbols global again, since they became part of official type signatures.